app/controllers/security_group_controller.rb

```diff
@@ -14,8 +14,6 @@ def self.display_methods
     %w(instances network_ports)
   end
 
-  menu_section :net
-
```

Comment:
Was there a way this functionality was supported prior to menu_section existing?  None of the euwe controllers use menu_section